sources/payment-hub/target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst

```diff
@@ -0,0 +1,132 @@
+/home/sg004/Code/gsoc-work/payment-hub/sources/payment-hub/src/main/java/hu/dpc/rt/psp/dto/gsma/PostalAddress.java
+/home/sg004/Code/gsoc-work/payment-hub/sources/payment-hub/src/main/java/hu/dpc/rt/psp/routebuilder/gsma/processor/PostTransactionsProcessor.java
+/home/sg004/Code/gsoc-work/payment-hub/sources/payment-hub/src/main/java/hu/dpc/rt/psp/component/RestClient.java
```

Comment:
Again, looks like a build script generated file which shouldn't be added to source control.